tensorflow/tools/pip_package/build_pip_package.sh

```diff
@@ -77,10 +102,14 @@ function main() {
 
   # protobuf pip package doesn't ship with header files. Copy the headers
   # over so user defined ops can be compiled.
-  mkdir -p ${TMPDIR}/google
-  rsync --include "*/" --include "*.h" --exclude "*" --prune-empty-dirs -a \
-    $RUNFILES/external/protobuf ${TMPDIR}/google
-  rsync -a $RUNFILES/third_party/eigen3 ${TMPDIR}/third_party
+  # TODO(pcloudy): rsync doesn't work well in msys on Windows
+  # Find an alternative for this on Windows
```

Comment:
Could a few simple `for header in $(find ... -name \*.h); do cp $header ...; done` loops work here?